dashboard/modules/job/job_consts.py

```diff
@@ -1,2 +1,13 @@
+import os
+
+# Job agent consts
+JOB_DIR = "{temp_dir}/job/{job_id}/"
```

Comment:
```suggestion
JOB_DIR_TEMPLATE = "{temp_dir}/job/{job_id}/"
```

Reply:
I think it's not necessary to add a _TEMPLATE suffix to all the const names.